### PR TITLE
Option colour in menu settings

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #14074

As identified in 14074 there is a class on all option fields for a component that adds the colour green to positive settings. However this class was not present for the same options when creating a menu item

### Testing Instructions
Create any menu item and on the options tab check to see that the value for Show or Yes are green when selected

### CODE REVIEW

File: components/com_search/views/search/tmpl/default.xml [hunk 1]
```diff
@@ -44,7 +45,7 @@
 				label="COM_SEARCH_FIELD_SEARCH_AREAS_LABEL"
 				description="COM_SEARCH_FIELD_SEARCH_AREAS_DESC"
 				useglobal="true"
-				>
```
Comment (Quy) [inline components/com_search/views/search/tmpl/default.xml]: Revert and move below.

File: components/com_content/views/archive/tmpl/default.xml [hunk 2]
```diff
@@ -167,8 +171,9 @@
 				label="JGLOBAL_LINK_CATEGORY_LABEL"
 				description="JGLOBAL_LINK_CATEGORY_DESC"
 				useglobal="true"
+				class="chzn-color"
 				>
-				<option value="0">JNo</option>
+				<option value="0">JNO</option>
```
Comment (Quy) [inline components/com_content/views/archive/tmpl/default.xml]: Capitalize `JYes`

File: components/com_content/views/categories/tmpl/default.xml [hunk 3]
```diff
@@ -520,8 +542,9 @@
 				label="JGLOBAL_LINK_CATEGORY_LABEL"
 				description="JGLOBAL_LINK_CATEGORY_DESC"
 				useglobal="true"
+				class="chzn-color"
 				>
-				<option value="0">JNo</option>
+				<option value="0">JNO</option>
```
Comment (Quy) [inline components/com_content/views/categories/tmpl/default.xml]: Capitalize `JYes`

File: components/com_content/views/categories/tmpl/default.xml [hunk 4]
```diff
@@ -542,8 +566,9 @@
 				label="JGLOBAL_LINK_PARENT_CATEGORY_LABEL"
 				description="JGLOBAL_LINK_PARENT_CATEGORY_DESC"
 				useglobal="true"
+				class="chzn-color"
 				>
-				<option value="0">JNo</option>
+				<option value="0">JNO</option>
```
Comment (Quy) [inline components/com_content/views/categories/tmpl/default.xml]: Capitalize `JYes`

File: components/com_content/views/categories/tmpl/default.xml [hunk 5]
```diff
@@ -562,8 +588,9 @@
 				description="JGLOBAL_LINK_AUTHOR_DESC"
 				label="JGLOBAL_LINK_AUTHOR_LABEL"
 				useglobal="true"
+				class="chzn-color"
 				>
-				<option value="0">JNo</option>
+				<option value="0">JNO</option>
```
Comment (Quy) [inline components/com_content/views/categories/tmpl/default.xml]: Capitalize `JYes`

File: components/com_content/views/category/tmpl/default.xml [hunk 6]
```diff
@@ -490,8 +514,9 @@
 				label="JGLOBAL_LINK_AUTHOR_LABEL"
 				description="JGLOBAL_LINK_AUTHOR_DESC"
 				useglobal="true"
+				class="chzn-color"
 				>
-				<option value="0">JNo</option>
+				<option value="0">JNO</option>
```
Comment (Quy) [inline components/com_content/views/category/tmpl/default.xml]: Capitalize `JYes`

File: components/com_content/views/featured/tmpl/default.xml [hunk 7]
```diff
@@ -240,6 +248,7 @@
 				label="JGLOBAL_LINK_CATEGORY_LABEL"
 				description="JGLOBAL_LINK_CATEGORY_DESC"
 				useglobal="true"
+				class="chzn-color"
```
Comment (Quy) [inline components/com_content/views/featured/tmpl/default.xml:251]: Capitalize `JYes` and `YNo` below.

File: components/com_content/views/featured/tmpl/default.xml [hunk 8]
```diff
@@ -299,6 +312,7 @@
 				label="JGLOBAL_LINK_AUTHOR_LABEL"
 				description="JGLOBAL_LINK_AUTHOR_DESC"
 				useglobal="true"
+				class="chzn-color"
```
Comment (Quy) [inline components/com_content/views/featured/tmpl/default.xml:315]: Capitalize `JYes` and `YNo` below.

File: components/com_newsfeeds/views/categories/tmpl/default.xml [hunk 9]
```diff
@@ -205,6 +215,7 @@
 				description="JGLOBAL_FILTER_FIELD_DESC"
 				default=""
 				useglobal="true"
+				class="chzn-color"
 				>
 				<option value="hide">JHIDE</option>
```
Comment (Quy) [inline components/com_newsfeeds/views/categories/tmpl/default.xml:220]: Change to `0`

Comment (brianteeman) [reply components/com_newsfeeds/views/categories/tmpl/default.xml:220]: Not sure - beyond the scope of this PR

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 5f0cde875651eadcfeff1ae358995d18e135a8ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18223">issues.joomla.org/tracker/joomla-cms/18223</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5f0cde875651eadcfeff1ae358995d18e135a8ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18223">issues.joomla.org/tracker/joomla-cms/18223</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanls

**Pull request merged**